docs/articles/documentation/reference/test-api/test/clientscripts.md

```diff
@@ -33,3 +33,97 @@ test
 > Relative paths are resolved against the test file location.
 
 You can use the [page](../../../guides/advanced-guides/inject-client-scripts.md#provide-scripts-for-specific-pages) option to specify pages into which scripts should be injected. Otherwise, TestCafe injects scripts into all pages visited during the test.
+
```

Comment:
To editors: no text was added or modified in this file except for the headers. Other changes are technical.